.rubocop_todo.yml

```diff
@@ -68,6 +69,22 @@ Lint/MissingCopEnableDirective:
     - 'app/classes/observation_report/raw.rb'
     - 'app/classes/observation_report/symbiota.rb'
 
+# Offense count: 16
+Lint/MissingSuper:
+  Exclude:
+    - 'app/classes/api/errors.rb'
+    - 'app/classes/api/ordered_range.rb'
+    - 'app/classes/api/upload.rb'
+    - 'app/classes/checklist.rb'
+    - 'app/classes/geocoder.rb'
+    - 'app/classes/pattern_search/errors.rb'
+    - 'app/classes/query/modules/active_record.rb'
+    - 'app/classes/query/modules/serialization.rb'
+    - 'app/models/language_exporter.rb'
+    - 'script/update_googlebots.rb'
+    - 'script/update_ip_stats.rb'
+    - 'test/uploaded_string.rb'
```

Comment:
I didn't understand how to handle these particular offenses.
In some cases, adding a call to `super` made the test suite throw errors.
In some cases it may have had no effect.
Here are two options:
- Leave `.rubocop.todo` as-is, create a Pivotal Story to fix the listed 16 listed offenses. 
- Remove the above from `.rubocop.todo`, and disable the cop within the code at each of the 16 listed places.

Reply:
I'm simply leaving `.rubocop_todo.yml` as-is.
That will prevent RuboCop complaining about the excluded files, but allow complaints about new offenses in other files. We can deal with the excluded files at our leisure. We'll probably end up disabling the cop in the source code or even globally. See also https://github.com/MushroomObserver/mushroom-observer/pull/655#issuecomment-683480839